src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/InputCrAsNewlineMultiline.javadoc

```diff
@@ -0,0 +1 @@
+* line 0
* Checks that commentContent starts with '*' javadoc comment identifier.
    *&lt;property name="ignorePattern" value="^ *\* *[^ ]+$"/&gt*
* This * is not a leading *
* @return true if commentContent starts with '*' javadoc comment identifier
* line 5
    * line 6

```

Comment:
Doesn't this file need to be added to the git attributes? Are you sure it didn't change the line endings?

Reply:
I think that you only need to set git attributes for windows line endings. git did not complain when I committed the file as it normally would with windows line endings. On fresh checkout:

```bash

➜  checkstyle git:(issue-10208) cat /home/nick/IdeaProjects/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/InputCrAsNewlineMultiline.javadoc
➜  checkstyle git:(issue-10208) cat -v /home/nick/IdeaProjects/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/InputCrAsNewlineMultiline.javadoc
* line 0^M* Checks that commentContent starts with '*' javadoc comment identifier.^M    *&lt;property name="ignorePattern" value="^ *\* *[^ ]+$"/&gt*^M* This * is not a leading *^M* @return true if commentContent starts with '*' javadoc comment identifier^M* line 5^M    * line 6^M%           

```

Reply:
Added both for good measure.